include/zephyr/bluetooth/gatt.h

```diff
@@ -981,6 +981,9 @@ struct bt_gatt_notify_params {
 	bt_gatt_complete_func_t func;
 	/** Notification Value callback user data */
 	void *user_data;
+#ifdef CONFIG_BT_EATT
+	enum bt_att_chan_option chan_option;
+#endif /* CONFIG_BT_EATT */
```

Comment:
It is guarded here to not change the API when EATT is disabled. If we are ok with changing the API also in that case, the guard can be removed.

I made this a bit field after your comment in the RFC (https://github.com/zephyrproject-rtos/zephyr/pull/41909#discussion_r787519694), so that it can more easily be expanded in the future.

Reply:
Yup :) As mentioned, it probably is fine as is, and it's easier to remove the guard in the future than it is change the type of this value.